ccm_web/client/src/components/Accordion.tsx

```diff
@@ -7,14 +7,17 @@ import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
 
 const useStyles = makeStyles((theme) => ({
   container: {
-    paddingTop: theme.spacing(1)
+    paddingTop: theme.spacing(1),
+    '& .MuiAccordionSummary-root': {
+      '&.Mui-focused, &.Mui-focusVisible': {
+        color: theme.palette.primary.main
+
+      }
+    }
   },
   summary: {
     backgroundColor: theme.palette.primary.main,
-    color: theme.palette.primary.contrastText,
-    '&:focus': {
-      color: theme.palette.primary.main
-    }
+    color: theme.palette.primary.contrastText
   },
```

Comment:
I'm not sure if you've heard from the UX team yet, but they may also want the expand icon to be darker when the focus is visible. That's a bit more complicated, since it doesn't actually have focus (its parent does).
<img width="600" alt="Screen Shot 2022-02-17 at 7 47 13 PM" src="https://user-images.githubusercontent.com/35741256/154598088-12b2bed4-5201-40a0-93da-e21350578304.png">

I'm not sure if there's a way to do this purely with CSS; there might be. I did find a way to do it with React + CSS. See my commits here: https://github.com/ssciolla/canvas-course-manager-next/tree/issue-328-on-focus-visible

Reply:
When I was looking at the elements CSS changes in the Chrome Dev tools  I saw as below i.e the background changing color to white at the MUI-summary component with both focused and focusVisible. So I Felt changing at container might work. Since container has access to all child elements.
 
I also saw in both focus and FocusVisible case the color change was occurring Hence I addressed both cases. I did [read](https://bharathvaj.me/blog/focus-vs-within-vs-visible) about focus and focusVisible what is difference they were more talking about mouse vs keyboard vs tap these might apply differently. So I felt I could apply to both. I wasn't sure if both was needed

<img width="662" alt="Screen Shot 2022-02-17 at 12 31 49 PM" src="https://user-images.githubusercontent.com/8579775/154690929-874ebdc0-b16f-40bd-aff3-2e5e5cd7a151.png">

I can apply at you changes at summary level instead of container.

Reply:
> I'm not sure if you've heard from the UX team yet, but they may also want the expand icon to be darker when the

I asked Janel to take a look before opening the PR, she did not comment if the icon needs a color change or not. So I left it.

>I'm not sure if there's a way to do this purely with CSS; there might be. I did find a way to do it with React + CSS.

Yeah.. I agree it needs to add a state, but I did play around a bit for changing the  color `ExpandIcon` has prop as `focused`, i applied that and I try to grab hold of `MuiSVGIcon` element, if I can change color on focus. But I did not.  So I wasn't sure if this was asked. So I just stopped.

But since you figured out a way to change the color for icon, we can make it happen. Can you open a PR against this change?

Reply:
Here's the PR. If the UX team likes the change, feel free to use it.